controllers/dspipeline_params.go

```diff
@@ -265,6 +269,8 @@ func (p *DSPAParams) SetupObjectParams(ctx context.Context, dsp *dspa.DataScienc
 		)
 		p.ObjectStorageConnection.Port = config.MinioPort
 		p.ObjectStorageConnection.Scheme = config.MinioScheme
+		p.ObjectStorageConnection.Secure = config.MinioSecure
```

Comment:
See above comment, we'll want to check if `.secure` is set here as well before setting some default.